<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Roman" name="Command line test" tests="1" failures="1" timestamp="9 Mar 2016 06:11:45 GMT" time="1.254" errors="0">
  <testcase name="testArea" time="0.008" classname="test.MathTests.AreaTest">
    <failure type="java.lang.AssertionError" message="expected [3] but found [25]">
      <![CDATA[java.lang.AssertionError: expected [3] but found [25]
	at test.MathTests.AreaTest.testArea(AreaTest.java:38)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- testArea -->
</testsuite> <!-- Command line test -->
